src/leap/bitmask/vpn/helpers/linux/bitmask-root

```diff
@@ -741,6 +742,11 @@ def firewall_start(args):
     # Must stay on 'top' of chain!
     if QUBES_PROXY and QUBES_VER >= 3 and run("grep", \
         "installed\ by\ " + SCRIPT, QUBES_FW_SCRIPT, exitcode=True) != 0:
+        if QUBES_VER == 4 and \
+            not os.path.isdir(os.path.dirname(QUBES_FW_SCRIPT)):
+            os.makedirs(os.path.dirname(QUBES_FW_SCRIPT))
+        if QUBES_VER == 3 and os.path.exists(QUBES_FW_SCRIPT):
+            os.rename(QUBES_FW_SCRIPT, QUBES_FW_SCRIPT + ".bak")
```

Comment:
It will overwrite only in the case that the current QUBES_FW_SCRIPT is not bitmask-generated (outer if block). But checking here as you suggest is also a good idea.